### PR TITLE
wallets & keys context menu

### PR DESCRIPTION
**Pull request by greg-schrammel**
Fixes BX-####
Figma link (if any):

## Screen recordings / screenshots

https://github.com/rainbow-me/browser-extension/assets/6232729/6bc875e5-e6eb-4cd2-b529-94e772b7cce9

### CODE REVIEW

File: src/entries/popup/pages/settings/privacy/walletsAndKeys/walletsAndKeys.tsx [hunk 1]
```diff
@@ -128,88 +241,87 @@ export const WalletsAndKeys = () => {
...
+            <WalletsAndKeysContextMenu key={idx} wallet={wallet}>
+              <Menu ref={firstNotBackedUp ? firstNotBackedUpRef : undefined}>
+                <MenuItem
+                  testId={`wallet-group-${idx + 1}`}
+                  first
+                  titleComponent={
+                    <MenuItem.Title
+                      text={`${i18n.t(
+                        wallet.type === KeychainType.HdKeychain
+                          ? 'settings.privacy_and_security.wallets_and_keys.recovery_phrase_label'
```
Comment (estebanmino) [inline src/entries/popup/pages/settings/privacy/walletsAndKeys/walletsAndKeys.tsx:253]: why not `{t('blabla')}` here?

File: src/entries/popup/pages/settings/privacy/walletsAndKeys/walletsAndKeys.tsx [hunk 2]
```diff
@@ -128,88 +241,87 @@ export const WalletsAndKeys = () => {
...
+                    <Inline alignHorizontal="center" alignVertical="center">
+                      <Button
+                        width="full"
+                        color="red"
+                        height="36px"
+                        variant="tinted"
+                        onClick={() => handleBackup({ wallet })}
+                      >
+                        {i18n.t(
+                          'settings.privacy_and_security.wallets_and_keys.back_up_now',
```
Comment (estebanmino) [inline src/entries/popup/pages/settings/privacy/walletsAndKeys/walletsAndKeys.tsx:317]: why not `{t('blabla')}` here?

**Review by estebanmino — APPROVED** [review]
lgtm

File: src/entries/popup/pages/settings/privacy/walletsAndKeys/walletsAndKeys.tsx (hunk 1 above)
Comment (greg-schrammel) [reply src/entries/popup/pages/settings/privacy/walletsAndKeys/walletsAndKeys.tsx:253]: idk I think I created the `t` after doing this and missed later, just updated both

**Review by derHowie — APPROVED** [review]

**Pull request merged**